cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt

```diff
@@ -101,6 +104,12 @@ class ClearlyDefinedUploadCommand : CliktCommand(
         help = "The ClearlyDefined server to upload to."
     ).enum<Server>().default(Server.DEVELOPMENT)
 
+    private fun logInnerError(errorBody: ResponseBody) {
+        val errorResponse = jsonMapper.readValue(errorBody.string(), ErrorResponse::class.java)
```

Comment:
Better would be `jsonMapper.readValue<ErrorResponse>(errorBody.string())`.

Reply:
I agree, but this requires `jackson-module-kotlin` as a dependency, which currently is not an (excplicit) dependency. However, I just realized not even `jackson-databind` is an explicit dependemcy, so it's leaking transitively. I'll have a look.

Do you think it's worth adding `jackson-module-kotlin` as a dependency for the syntactic sugar?

Reply:
Done in a new commit, as it was like that before my changes.

Reply:
I'd say yes, because we also use it in the other modules, and a few lines below we use our own extension function which uses a similar syntax. But it's not really related to this PR, because you just moved the line.